app/models/forms/work.rb

```diff
@@ -0,0 +1,150 @@
+module Forms
+  class Work < Base
```

Comment:
Work is kinda complicated because it needs to handle both the work and work version models. It also has multiple nested forms (authors, contributors, email). This is an example of where a parent model is used.